niworkflows/utils/spaces.py

```diff
@@ -165,12 +167,32 @@ def __attrs_post_init__(self):
                 spec["den"] = FSAVERAGE_DENSITY[space]
                 object.__setattr__(self, "spec", spec)
 
+        # XXX: This won't cover dhcpAsym, dhcpSym, or onavg
         if self.space.startswith("fs"):
             object.__setattr__(self, "dim", 2)
 
+        if "volspace" in self.spec:
```

Comment:
This check would hopefully only be necessary in the short-term, since mixed standard/non-standard CIFTIs should be allowed.